vite.config.ts

```diff
@@ -1,7 +1,15 @@
-import { defineConfig } from 'vite'
-import react from '@vitejs/plugin-react'
+/* eslint-disable import/no-extraneous-dependencies */
+/// <reference types="vitest" />
```

Comment:
remove eslint disable no need to this you need to be good with eslint in other words change it all